src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -43,34 +47,44 @@ export default function ConfirmedAppointmentDetailsPage() {
   const featureBreadcrumbUrlUpdate = useSelector(state =>
     selectFeatureBreadcrumbUrlUpdate(state),
   );
+  const featureFetchClaimStatus = useSelector(
+    selectFeatureTravelPayViewClaimDetails,
```

Comment:
I wonder if there's two toggles because they could show claim status for existing claims but not have the ability to submit claims. 🤔 That would make sense. I'll ask in the travel pay channel.

Reply:
> there's two toggles because they could show claim status for existing claims but not have the ability to submit claims

Yes, that's exactly the case. I'm not sure which you're working on right now (the claim status or the submit claims) which is why I'm not sure which flag you'll need right now, but yes, we might enable the "view claim details" flag before the "submit for mileage" flag, so ideally both would be in place when it's time for the features to be flipped.

Reply:
This is just for modifying the fetch for claim status the next ticket will be the UI logic and require both flags. Thanks for the clarification!